sigar_linux_test.go

```diff
@@ -287,7 +287,15 @@ memory2 /smart/fox/jumped/by/lazy/dog/duplicate cgroup2 irrelevant,options
 			It("fails for missing files", func() {
 				limit, err := determineMemoryLimit(``)
 				Expect(err).To(HaveOccurred())
-				Expect(err.Error()).To(Equal("open " + procd + "/memory/memory.limit_in_bytes: no such file or directory"))
+				// it will falls back to memory.stat when memory.limit_in_bytes not found
```

Comment:
this is already the case that both files are missing

Reply:
Yeah, you're absolutely correct. I don't know what I was thinking at the time, and wish I had left more detailed notes so that I would have some idea.